arroyo-worker/src/connectors/kafka/source/mod.rs

```diff
@@ -131,10 +135,11 @@ where
             .set("enable.auto.commit", "false")
             .set(
                 "group.id",
-                format!(
+                self.group_id.unwrap_or(format!(
```

Comment:
This is a nit since this isn't a performance-sensitive part of the code, but generally the closure form `upwrap_or_else` is prefered to `unwrap_or` when the None case is going to do something expensive like call format.

```suggestion
                self.group_id.unwrap_or_else(|| format!(
```